tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -540,8 +540,12 @@ EOF
 
 	if [ -f "$chrony_systemd_service" ]; then
 		# Remove user option, user could not exist in the rootfs
+		# Set the /var/lib/chrony for ReadWritePaths to be ignored if
+		# its nonexistent, this broke the service on boot previously
+		# due to the directory not being present "(code=exited, status=226/NAMESPACE)"
 		sed -i -e 's/^\(ExecStart=.*\)-u [[:alnum:]]*/\1/g' \
-		       -e '/^\[Unit\]/a ConditionPathExists=\/dev\/ptp0' ${chrony_systemd_service}
+		       -e '/^\[Unit\]/a ConditionPathExists=\/dev\/ptp0' \
+					 -e '/^ReadWritePaths=(.+) \/var\/lib\/chrony (.+)$/ReadWritePaths=\1 -\/var\/lib\/chrony \2/' ${chrony_systemd_service}
```

Comment:
It seems the indentation is not aligned (two tabs + some spaces vs only tabs on your patch).

Apart from this, this looks good to me.
If the issue is happening with the default clearlinux images, I wonder how we could report this back inside Intel.

Reply:
I think because the Clear Linux base image doesn't have `/var` writable (correct me if I'm wrong) this configuration is needed. It isn't something we'd try to account for by default in the distro though so making the change here seems best.

Reply:
This used to work in the past, so I am wondering what exactly has caused this breakage. Has there been a change on the Clearlinux side or the chrony service?

Reply:
Looks like the indentation is fixed now